### PR TITLE
Add alignment, quotations, spacing. Also fix the link to the style guide.

### PR DESCRIPTION
**Pull request by BJordi**
## Summary

Add following items to the Vue Style Guide:
- Alignment
- Quotes
- Spacing

### CODE REVIEW

File: frontend/vue/style-guide.md [hunk 1]
```diff
@@ -115,7 +124,7 @@ As Vue separates the template from the script, we're going to follow the convent
 ```pug
 // bad
 <template lang="pug">
-  my-component(greetingText='hi')
+  my-component(greetingText="hi")
```
Comment (rsuescun) [inline frontend/vue/style-guide.md:127]:
```suggestion
  my-component(greetingText='hi')
```

File: frontend/vue/style-guide.md [hunk 2]
```diff
@@ -128,7 +137,7 @@ As Vue separates the template from the script, we're going to follow the convent
 ```pug
 // good
 <template lang="pug">
-  my-component(greeting-text='hi')
+  my-component(greeting-text="hi")
```
Comment (rsuescun) [inline frontend/vue/style-guide.md:140]:
```suggestion
  my-component(greeting-text='hi')
```

File: frontend/vue/style-guide.md [hunk 3]
```diff
@@ -220,3 +229,135 @@ my-component(
...
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.container {
+  font-family: "Lato";
+}
+</style>
```
Comment (rsuescun) [inline frontend/vue/style-guide.md:328]:
```suggestion
<template lang='pug'>
.container
  input(v-model="name" type="filters.name")
<template>

<script>
export default {
  data() {
    return {
      filters: {
        name: ""
      }
    }
  }
}
</script>

<style lang='scss' scoped>
.container {
  font-family: "Lato";
}
</style>
```

File: frontend/vue/style-guide.md [hunk 4]
```diff
@@ -220,3 +229,135 @@ my-component(
...
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.container {
+  font-family: 'Lato';
+}
+</style>
```
Comment (rsuescun) [inline frontend/vue/style-guide.md:354]:
```suggestion
<template lang="pug">
.container
  input(v-model='name' type='filters.name')
<template>

<script>
export default {
  data() {
    return {
      filters: {
        name: ''
      }
    }
  }
}
</script>

<style lang="scss" scoped>
.container {
  font-family: 'Lato';
}
</style>
```

File: frontend/vue/style-guide.md [hunk 5]
```diff
@@ -220,3 +229,135 @@ my-component(
...
+        { value: element.id, label: element.name }
+      ))
+    }
+  }
+}
+```
+
+## Quotes
+
+In the template and html tags, always use double quotes (`"`) as the main quotation mark, and single ones (`'`) as secondary. On the contary, in the script and the style sections use single quotes as the main quotation mark and double as secondary.
```
Comment (rsuescun) [inline frontend/vue/style-guide.md:303]:
```suggestion
In the template, script and style sections content, always use single quotes (' ') as the main quotation mark, and double quotes (" ") as secondary. On the contrary, in the `template` and `style` tags use double quotes.
```

**Comment by mcallegari10** [comment]
@BJordi The problem is that you are saying we use single quotes, but in the styleguide you used double quotes instead of single. I agree on your point about double and singles, so change it in the styleguide please.

File: frontend/vue/style-guide.md [hunk 6]
```diff
@@ -6,11 +6,20 @@
...
+5. [Alignment](#alignment)
+6. [Quotes](#quotes)
+7. [Spacing](#spacing)
+8. [Vue-Router](#vue-router)
 
 ## Basic rules
 
 * When using the data property on a component, it **must** be a function that return an object. The only exception to the rule is in a root Vue instance, since only a single instance will ever exist.
 
+* Always use v-bind shortcut `:` instead of the whole directive `v-bind:`. E.g.: Do `:value="someValue"` instead of `v-bind:value="someValue"`.
```
Comment (FrankIglesias) [inline frontend/vue/style-guide.md:18]: missing `@click` to replace v-on:click

**Comment by BJordi** [comment]
@FrankIglesias Sorry, never saw your comment.

Added in this PR: 
> * Empty line between template, script, style tag.
> * Tag order.
> * Prop default value definition
> * Property definition with camelCase and passing properties with kebab-case

Already added in the style guide:
> * Component attributes (data, props, components mounted, methods, etc) order
> * Folder structure inside a project.

Will be added in next PRs:
> * Max line rule
> * When to indent properties or not? (I think it depends on the rule above)
> * Usage of name attribute inside components
> * When is preferred to use functional components

Avoid? I'm not getting what you're trying to say with these:
> * Why we should avoid the v-show property
> * Vue definition for methods that call for async functions (like API requests)

I think the next topics haven't to be with the purpose of this document. It's not a framework documentation, API or guide. It's a style guide, which is not the same. If you want, we can add it, but then we'll be giving this documentation a larger scope and not the one of simply a "style guide".
> * How to avoid using ternaries or && with classes
> * Data definition (function or arrow)
> * When during lifecycle hooks you should call api requests.

**Review by rsuescun — APPROVED** [review]

**Pull request merged**